vassal-app/src/main/java/VASSAL/build/module/map/BoardPicker.java

```diff
@@ -176,9 +177,7 @@ public void addTo(Buildable b) {
     for (final Board board : possibleBoards) {
       board.setMap(map);
     }
-    if (b instanceof Translatable) {
-      getI18nData().setOwningComponent((Translatable) b);
-    }
+    getI18nData().setOwningComponent((Translatable) b);
```

Comment:
Isn't this going to throw a ClassCastException? There's no guarantee that every Buildable is a Translatable.

Reply:
Yes indeed. I fell for Intellij again that insists this is OK, but I agree, While most Buildables end up implementing Translatable, Buildable itself does not and the potential for a Class Cast Exception is there. Reverted.